a10_octavia/controller/worker/controller_worker.py

```diff
@@ -307,7 +305,7 @@ def create_load_balancer(self, load_balancer_id):
         topology = CONF.a10_controller_worker.loadbalancer_topology
 
         store[constants.UPDATE_DICT] = {
-            constants.LOADBALANCER_TOPOLOGY: topology
+            constants.TOPOLOGY: topology
```

Comment:
This change is incorrect. It is expecting the string `loadbalancer_topology` not `topology` to be the key.